python/ray/autoscaler/_private/prom_metrics.py

```diff
@@ -74,7 +74,7 @@ def __init__(
                 "time of 3 minutes will be observed 8 times.",
                 labelnames=("SessionName",),
                 unit="seconds",
-                namespace="autoscaler",
+                namespace="ray_autoscaler",
```

Comment:
Can we only change the namespace of the 3 metrics we need and not include "autoscaler" in them since they aren't necessarily autoscaler related? 

Ideally we audit the rest of the metrics as we move them into the Ray namespace (I think many of them aren't very well thought out).